### PR TITLE
fix(trading): order stopped intent toasts

### LINKED ISSUE #3238: Add post only and reduce only order options to deal ticket
## Story

As a trader
I want to be able to place post only and reduce only orders (aka execution flags or order conditions)
So that can minimize risk and maximise profit

## Acceptance Criteria

- [ ] I can place a post only order
- [ ] I can place a reduce only order

more detailed ACs here: https://github.com/vegaprotocol/specs/pull/1640

## Tasks

- [x] Explore and sketch
- [ ] Team and stakeholder review
- [ ] Visual Design
- [ ] Team review
- [ ] Dev
- [ ] Test

### Changes to deal ticket

- Add two check boxes to the deal ticket: one for post only one for reduce only
- Add logic so that only one of these can be selected at the same time. (Radio buttons were considered instead but Checkboxes are more appropriate)
- Possibly a later ticket: add tool tips on hover of each with an explanation of what they mean

Suggested draft content

> Selecting Post only will ensure the order is not filled immediately but is placed on the book as a passive order. When the order is processed it is either be stopped (if it would be filled immediately) Or placed in the book as a passive order until and price taker matches with it

> Selecting Reduce only will ensure that this order will not increase the size of an open position. When the order is matched it will only trade enough volume to bring your open volume towards 0 but never change the direction of your position. If applied to a limit order that is not instantly filled the order will be stopped.

- Link to documentation from these tool tips: https://github.com/vegaprotocol/documentation/pull/544

- When the TIF is FOC or IOC the Post only check box should be unchecked and disabled 
-The tool tip when hovering over the disabled check box should tell you that this can not be used for the selected TIF

> "Post only" can not be used on "Fill or Kill" or "Immediate-or-canceled" orders

- The selected state of the Execution flags should be remembered as part of the general deal ticket state (size value etc). This includes the change to state that happen when the box is unchecked automatically.

### Changes to Orders table

Image

- Add any selected execution flag to the TIF column. Simply append the flag to the end of the time in force text. e.g. "Good til' close. Reduce only" or "Good til` time" if no execution flag is applied.
- ensure that the "[reason](https://docs.vega.xyz/mainnet/api/graphql/enums/order-rejection-reason)" is being displayed in the status if there is one. I know this is the case for rejected orders but not sure if it has been implemented for Stopped orders too. background:  "...it is instead immediately Stopped with a reason informing the trader that the order was stopped to avoid a trade occurring" [source](https://github.com/vegaprotocol/specs/blob/master/protocol/0014-ORDT-order_types.md#execution-flags).

### Changes to toast

none

### 

## Additional details / background info

Specs PR: https://github.com/vegaprotocol/specs/pull/1625
Related core issue: https://github.com/vegaprotocol/vega/issues/7851
Related core pull request: https://github.com/vegaprotocol/vega/pull/7852

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #3238 

# Description ℹ️

- [x] Fix order intent and rejection error
- [x] Add order list label in TIF

# Demo 📺

image
image

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

**Review by jeremyletang — APPROVED** [review]

File: libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx [hunk 1]
```diff
@@ -179,6 +185,8 @@ export const DealTicket = ({
   // if an order doesn't exist one will be created by the store immediately
   if (!order || !normalizedOrder) return null;
 
+  console.log('errors', errors);
```
Comment (bglownia) [inline libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx:188]: console.log

File: libs/accounts/src/lib/transfer-form.tsx [hunk 2]
```diff
@@ -232,25 +232,43 @@ export const TransferFee = ({
...
+        </div>
       </div>
-      <div
-        data-testid="transfer-fee"
-        className="text-neutral-500 dark:text-neutral-300"
-      >
-        {value}
+      <div className="flex justify-between gap-4 items-center flex-wrap">
+        <Tooltip
+          description={t(
```
Comment (bglownia) [inline libs/accounts/src/lib/transfer-form.tsx]: Do not translate variables
`t("The total amount plus the transfer fee, which is set by the network parameter %s", feeFactor)`

File: libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx (hunk 1 above)
Comment (MadalinaRaicu) [reply libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx:188]: thanks will remove now testing some undefined rejection reasons

File: libs/orders/src/lib/components/order-list/order-list.tsx [hunk 3]
```diff
@@ -218,7 +218,14 @@ export const OrderListTable = memo(
                 return `${Schema.OrderTimeInForceMapping[value]}: ${expiry}`;
               }
 
-              return value ? Schema.OrderTimeInForceMapping[value] : '';
+              const tifLabel = value
+                ? Schema.OrderTimeInForceMapping[value]
+                : '';
+              const label = `${tifLabel}${data?.postOnly ? '. Post Only' : ''}${
+                data?.reduceOnly ? '. Reduce only' : ''
```
[the remark this answers is not in the document]
Comment (MadalinaRaicu) [reply libs/orders/src/lib/components/order-list/order-list.tsx:226]: true, updating it now

File: libs/accounts/src/lib/transfer-form.tsx (hunk 2 above)
Comment (MadalinaRaicu) [reply libs/accounts/src/lib/transfer-form.tsx]: removed this change

File: libs/orders/src/lib/components/order-list/order-list.tsx [hunk 4]
```diff
@@ -218,7 +219,14 @@ export const OrderListTable = memo(
                 return `${Schema.OrderTimeInForceMapping[value]}: ${expiry}`;
               }
 
-              return value ? Schema.OrderTimeInForceMapping[value] : '';
+              const tifLabel = value
+                ? Schema.OrderTimeInForceMapping[value]
```
Comment (mattrussell36) [inline libs/orders/src/lib/components/order-list/order-list.tsx:223]: chore (blocking): This looks incorrect. This is the status column we should be using the status mapping

Comment (mattrussell36) [reply libs/orders/src/lib/components/order-list/order-list.tsx:223]: oh nvm my mistake

File: libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx [hunk 5]
```diff
@@ -158,6 +158,12 @@ export const DealTicket = ({
     return disabled;
   }, [order]);
 
+  useEffect(() => {
```
Comment (mattrussell36) [inline libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx:161]: chore(blocking): we remove this effect and instead move this logic to the TIF callback function.

Comment (MadalinaRaicu) [reply libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx:161]: I did this initially but it's not enough. 
Whenever it gets disabled we need to make sure postOnly gets false

Comment (MadalinaRaicu) [reply libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx]: updated again reduceOnly

**Review by mattrussell36 — APPROVED** [review]

**Pull request merged**